mypy/checkmember.py

```diff
@@ -109,10 +109,17 @@ def analyze_member_access(name: str,
     elif isinstance(typ, NoneTyp):
         if chk.should_suppress_optional_error([typ]):
             return AnyType(TypeOfAny.from_error)
-        # The only attribute NoneType has are those it inherits from object
-        return analyze_member_access(name, builtin_type('builtins.object'), node, is_lvalue,
-                                     is_super, is_operator, builtin_type, not_ready_callback, msg,
-                                     original_type=original_type, chk=chk)
+        if name == '__bool__':
+            # The only attribute of NoneType that is not inherited from object.
```

Comment:
The position and content of this comment should be updated.

Reply:
> The position and content of this comment should be updated.

I've updated the position and content. Hopefully for the better. :sweat_smile: